src/renderer/components/dialogs/ViewProfile.tsx

```diff
@@ -30,16 +32,31 @@ import { selectChat } from '../helpers/ChatMethods'
 const ProfileInfoName = ({
   name,
   address,
+  lastSeen,
 }: {
   name: string
   address: string
+  lastSeen: number
 }) => {
+  const tx = useTranslationFunction()
+  let lastSeenString = ''
+
+  // Dates from 1970 mean that contact has never been seen
+  if (lastSeen == 0) {
+    lastSeenString = tx('last_seen_unknown')
+  } else {
+    const extended = false
+    const date = formatRelativeTime(lastSeen * 1000, { extended })
```

Comment:
```suggestion
    const date = formatRelativeTime(lastSeen * 1000, { extended: false })
```

Reply:
One last small code nitpick and then I'm happy to merge :P

Reply:
Done! :D